app/models/story.rb

```diff
@@ -5,22 +5,43 @@
 class Story
   include Mongoid::Document
   include Mongoid::Timestamps
+  include AASM
 
   belongs_to :ore_aggregation, class_name: 'ORE::Aggregation', inverse_of: :story,
                                autobuild: true, index: true, dependent: :destroy,
                                touch: true
   belongs_to :created_by, class_name: 'User', optional: true, inverse_of: :stories,
                           index: true
 
+  field :aasm_state
+
   index(created_at: 1)
   index(updated_at: 1)
+  index(aasm_state: 1)
 
   accepts_nested_attributes_for :ore_aggregation
 
   validates_associated :ore_aggregation
 
   delegate :to_rdf, :rdf_graph_to_rdfxml, to: :ore_aggregation
 
+  aasm do
+    state :draft, initial: true
+    state :published, :deleted
+
+    event :publish do
+      transitions from: :draft, to: :published
+    end
+
+    event :unpublish do
+      transitions from: :published, to: :draft
+    end
+
+    event :wipe do # named :wipe and not :delete because Mongoid::Document brings #delete
+      transitions from: :draft, to: :deleted
+    end
+  end
+
   rails_admin do
     list do
       field :ore_aggregation
```

Comment:
let's add `field :aasm_state` to the list and show sections already.

Perhaps also to the edit view if it's possible to lock it down to only admin role users. We can leave this for the ticket that implements changing states though.